webform_civicrm.install

```diff
@@ -302,3 +302,27 @@ function webform_civicrm_update_8003() {
     $webform->save();
   }
 }
+
+/**
+ * Set Contact 1 as the assigned for Contributions in all webforms
+ * (It keeps default behavior <= 6.2.0)
+ */
+function webform_civicrm_update_8004() {
+  \Drupal::service('civicrm')->initialize();
+  $utils = \Drupal::service('webform_civicrm.utils');
+  $webforms = Webform::loadMultiple();
+  foreach ($webforms as $webform) {
+    $handler = $webform->getHandlers('webform_civicrm');
+    $config = $handler->getConfiguration();
+    if (empty($config['webform_civicrm'])) {
+      continue;
+    }
+    $settings = &$config['webform_civicrm']['settings'];
+    // If Contribution enabled, set Contact 1 assigned as default behavior <= 6.2.0
+    if ($settings['civicrm_1_contribution_1_contribution_enable_contribution'] == "1") {
+      $settings['civicrm_1_contribution_1_contribution_contact_id'] = 1;
```

Comment:
@sluc23 I think you also need to add it to the $settings['data']? Something like -

    $settings['data']['contribution'][1]['contribution'][1]['contact_id'] = 1;

Reply:
@jitendrapurohit  added the missing params and tested
The test:
- Installed webform_civicrm with a previous release. Enabled contribution tab.
- Updated the webform_civicrm with ixiam:6.x#DevAssingContribs, executed the updb. Made a submit into webform. Verified that the contribution was assigned correctly to the contact.

Thx for the previous advice!